docs/tools/conf.py

```diff
@@ -96,7 +96,7 @@
 
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
-source_suffix = [".rst", ".md"]
```

Comment:
Generated this message in the doc build:

Converting `source_suffix = ['.rst', '.md']` to `source_suffix = {'.rst': 'restructuredtext', '.md': 'restructuredtext'}`.